WrightTools/artists/_base.py

```diff
@@ -463,6 +464,100 @@ def imshow(self, *args, **kwargs):
             super().invert_yaxis()
         return out
 
+    def scatter(self, *args, **kwargs):
+        """Scatter plot a channel against two _variables_.
+        Scatter point color reflects channel values.
+        Data need not be structured.
+        If data object is not provided, scatter reverts to the [matplotlib parent method](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html?highlight=scatter#matplotlib.axes.Axes.scatter)
```

Comment:
```suggestion
        If data object is not provided, scatter reverts to the `matplotlib parent method <https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html?highlight=scatter#matplotlib.axes.Axes.scatter>`_.
```

Docstrings use [rst links](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks), not markdown links